src/goose/cli/session.py

```diff
@@ -239,6 +245,55 @@ def _log_cost(self) -> None:
         get_logger().info(get_total_cost_message(self.exchange.get_token_usage()))
         print(f"[dim]you can view the cost and token usage in the log directory {LOG_PATH}")
 
+    def _prompt_overwrite_session(self) -> None:
+        print(f"[yellow]session already exists at {self.session_file_path}.[/]\n")
+        print("would like to overwrite the existing session?")
+        while True:
```

Comment:
nit: rich prompt encapsulates encapsulates the retry/invalid choice handling pretty well for this

Reply:
ahh, good catch! glossed over that detail in the docs

Reply:
[aa289ca](https://github.com/block-open-source/goose/pull/101/commits/aa289ca876b1944f501f408d187806c6c656f72b)